src/components/pages/HomePage.tsx

```diff
@@ -20,6 +20,14 @@ import { hooks } from '@/config/queryClient';
 import ItemLink from '../common/ItemLink';
 import ItemLoaderSkelton from '../common/ItemLoaderSkeleton';
 
+const HomePageWrapper = ({ children }: { children: JSX.Element }) => {
+  return (
+    <Box p={2}>
+      <Container>{children}</Container>
+    </Box>
+  );
+};
```

Comment:
Oupps sorry I think I told you wrong. I meant to have it like:
```suggestion
const HomePageWrapper = () => {
  return (
    <Box p={2}>
      <Container>
        <HomePage/>
      </Container>
    </Box>
  );
};
```

And you can remove the `<HomePageWrapper>` from inside the `HomePage` component.
Lastly you have to export the HomePageWrapper instead of the Homepage component.